addons/web/static/src/legacy/js/views/basic/basic_model.js

```diff
@@ -336,7 +336,7 @@ var BasicModel = AbstractModel.extend({
                     return false;
                 }
                 // 2.2. no drop on new addition on "UPDATE"
-                var lastEntry = _.last(parent._savePoint);
+                var lastEntry = parent._savePoint.slice(-1)[0];
```

Comment:
```suggestion
                var lastEntry = parent._savePoint.at(-1);
```

We can use `at` but the support for iOS is only `15.4`